posthog/queries/base.py

```diff
@@ -207,18 +207,9 @@ def filter_persons(team_id: int, request: request.Request, queryset: QuerySet) -
         uuids = request.GET["uuid"].split(",")
         queryset = queryset.filter(uuid__in=uuids)
     if request.GET.get("search"):
-        parts = request.GET["search"].split(" ")
-        contains = []
-        for part in parts:
-            if ":" in part:
-                matcher, key = part.split(":")
-                if matcher == "has":
-                    # Matches for example has:email or has:name
-                    queryset = queryset.filter(properties__has_key=key)
-            else:
-                contains.append(part)
         queryset = queryset.filter(
-            Q(properties__icontains=" ".join(contains)) | Q(persondistinctid__distinct_id__icontains=" ".join(contains))
+            Q(properties__icontains=request.GET.get("search"))
```

Comment:
super nit: we seem to use a pattern of `get` to check if it exists and then just access it directly inside the conditional.